cluster/src/main/scala/org/apache/spark/executor/SnappyExecutor.scala

```diff
@@ -178,6 +176,30 @@ class SnappyExecutor(
     }
   }
 
+  def removeJarsFromExecutorLoader(jars: Array[String]): Unit = {
+    synchronized {
+      jars.foreach(name => {
+        val localName = name.split("/").last
+        var jarFile = new File(SparkFiles.getRootDirectory(), localName)
+        if (jarFile.exists()) {
+          jarFile.delete()
```

Comment:
For DEPLOY JAR, this deletes the jar file copied to executor/server's work directory.
What is the behavior for DEPLOY PACKAGE?

Reply:
For deploy package also, this deletes the jar file copied to executor/servers work directory